src/components/Work/ParentCollections.js

```diff
@@ -5,42 +5,44 @@ import ThisItem from "./ThisItem";
 import SectionTop from "../UI/SectionTop";
 import { reactiveSearchFacets } from "../../services/reactive-search";
 
-const ParentCollections = ({ adminSetItems = [], collection, item }) => {
-  const adminSetTitle = item.admin_set ? item.admin_set.title[0] : "";
+const ParentCollections = ({ libraryUnitItems = [], collection, item }) => {
+  const libraryDepartment = item.administrativeMetadata.libraryUnit
+    ? item.administrativeMetadata.libraryUnit.label
+    : "";
   return (
     <div>
-      {item && adminSetItems.length > 0 && (
+      {item && libraryUnitItems && libraryUnitItems.length > 0 && (
         <section className="section" data-testid="section-library-department">
           <SectionTop
             sectionTitle="Library Department"
-            optionalSubhead={adminSetTitle}
+            optionalSubhead={libraryDepartment}
             optionalButtons={[
               {
                 label: "View All Items in Library Department",
                 url: "/search",
                 state: {
                   facet: reactiveSearchFacets.find(
-                    facet => facet.value === "LibraryDepartment"
+                    (facet) => facet.value === "LibraryDepartment"
                   ),
-                  searchValue: adminSetTitle
-                }
-              }
+                  searchValue: libraryDepartment,
+                },
+              },
             ]}
           />
-          <PhotoGrid items={adminSetItems} hideDescriptions={true} />
+          <PhotoGrid items={libraryUnitItems} hideDescriptions={true} />
         </section>
       )}
 
-      {item && collection.items.length > 0 && (
+      {item && collection.items && collection.items.length > 0 && (
```

Comment:
If this component shouldn't do anything without an `item` value, then you could do a short-circuit `return` in the first line of the component code, and tighten up the component code a bit.

```
if (!item) return;
```